R/zchunk_LA131.enduse.R

```diff
@@ -1,6 +1,8 @@
 #' module_energy_LA131.enduse
 #'
-#' Briefly describe what this chunk does.
+#' This chunk generates the following two tables:
+#'      Final scaled energy input by GCAM region / end-use sector, incl CHP / fuel / historical year; and
+#'      Share of heat consumption by end-use sector within GCAM region / historical year
 #'
```

Comment:
See `{item}` syntax I added to your other PRs and use here if possible--makes the rendered help page much easier to read.